src/Model/Model.ts

```diff
@@ -306,6 +307,10 @@ export class Model {
     return Object.keys(this.$dirty()).length > 0;
   }
 
+  public toJSON(): any {
```

Comment:
```suggestion
  public toJSON(): unknown {
```

Reply:
Will do that! Just one question. How should I change my test to still make it works. Because I can't access properties on an unknown object. Should I cast it for example? Sorry, my Typescript's knowledge is not so good.

Reply:
I suggest to change the test to use `JSON.stringify(obj)` and assert the contents of the returned string. This is why we are implementing toJSON after all :)

Reply:
True, good idea thanks!